### DISCUSSION TITLE
Some one please help me I’m having trouble with the codes

### DISCUSSION THREAD

**Discussion by Scotty1luv** [community/community#50957] [Copilot Conversations]
How do you do this stuff

**Comment by CodeMaster7000** [comment]
@Scotty1luv Could you specify what you are having trouble with?

**Reply by Scotty1luv** [reply]
Yes I will but first thank you for responding to me. For starters I'm having trouble getting started I'm
Not quite sure of what I'm doing. The terminology is all foreign to me, I'm not a tech savvy individual but I'm trying to grasp all of it and put it to some use. The term issues I think I get but I'm not able to figure it out. License yes I get it but putting it on my work not easy. So basically I'm fish out of water. Thank you again look forward to your help.

**Reply by LukeSavefrogs** [reply]
> Yes I will but first thank you for responding to me. For starters I'm having trouble getting started I'm Not quite sure of what I'm doing. The terminology is all foreign to me, I'm not a tech savvy individual but I'm trying to grasp all of it and put it to some use. The term issues I think I get but I'm not able to figure it out. License yes I get it but putting it on my work not easy. So basically I'm fish out of water. Thank you again look forward to your help.

What?
You're not saying what do you need help with...

- Github? Copilot? Git? Something else?
- Have you tried searching online for simple tutorials?

**Comment by Scotty1luv** [comment]
Yes still watching I’m getting it slowly

**Comment by CodeMaster7000** [comment]
@Scotty1luv Hopefully the below information can help you:

The idea of licenses is to protect your work from being stolen by others. There are various licenses such as Apache and MIT (you can learn more about these by reading the license document). When it comes to applying a license, create a file named "LICENSE", select a license template, and add your name and copyright year to it.

Other terms you might come across are issues and pull requests. You can create an issue to raise awareness about a bug in the code and ask people to fix it. If you wish to make a change to a branch you do not have write access to, you can create a pull request and a moderator will review it. He/she will either close it (your changes were denied) or merge it (your changes have been implemented).

Hopefully, this helps you in getting to grips with the basics of GitHub. Check out GitHub docs or online tutorials if you need any further help.

**Comment by SmashedFrenzy16** [comment]
Hi @Scotty1luv,

Issues on GitHub Repositories are a place where you can submit a bug or an issue in the code that you want to be fixed. 

Licenses are files that help your code become yours and make your code unable to be stolen with the right license. The Apache 2.0 license can be used for code that you want to be more protected, and the MIT license for code that is not so important but still needs to be protected. You can create a license by going into the license drop down menu when creating a repository. This link should help you further for licenses if you need it: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

Hope this helps!

Please also mark this as an answer as I am trying to get the Galaxy Brain Achievement on GitHub. Thank you!
